pytest.ini

```
@@ -6,5 +6,7 @@ flake8-ignore =
     setuptools/py*compat.py F811
 doctest_optionflags=ELLIPSIS ALLOW_UNICODE
 filterwarnings =
-	# https://github.com/pypa/setuptools/issues/1823
-	ignore:bdist_wininst command is deprecated
+    # Fail on warnings
+    error
+    # https://github.com/pypa/setuptools/issues/1823
+    ignore:bdist_wininst command is deprecated
```

Shouldn't this be something like 
```suggestion
    ignore:bdist_wininst command is deprecated:DeprecationWarning
```

---

It works either way. It seems to me unnecessary to additionally specify `DeprecationWarning` unless we could imagine that a warning that's not a `DeprecationWarning` would be used to indicate "bdist_wininst command is deprecated" and that we would want to trap _that_ warning but not this one.

---

Plus, this PR isn't concerned with that line; it's just getting reindented to follow the convention I use elsewhere.